update_dockerfiles_on_merge.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+
+source common_functions.sh 
+
+REMOTE_UPSTREAM="upstream"
+BRANCH_NAME="automated-branch-for-dockerfiles"
+
+function proceed_to_update() {
+    UPSTREAM=$(cat .git/config | grep -m 1 -B 1 "https://github.com/AdoptOpenJDK/openjdk-docker" | head -n 1 | cut -d '"' -f 2)
+    if [ -z $UPSTREAM ]; then
+        if cat .git/config | grep -Fxq "[remote \"upstream\"]"; then
+            git remote add temp-upstream https://github.com/AdoptOpenJDK/openjdk-docker
+            REMOTE_UPSTREAM="temp-upstream"
+        else
+            git remote add upstream https://github.com/AdoptOpenJDK/openjdk-docker
+        fi
+    else
+        REMOTE_UPSTREAM="$UPSTREAM"
+    fi
+    git checkout master
+    git fetch ${REMOTE_UPSTREAM}
+    git merge ${REMOTE_UPSTREAM}/master
+    local CHECK_LOCAL=$(git branch --list ${BRANCH_NAME})
+    if [[ -z ${CHECK_LOCAL} ]]; then
+        git checkout -b ${BRANCH_NAME}
+    else
+        git branch -D ${BRANCH_NAME}
+        git checkout -b ${BRANCH_NAME}
+    fi
+    source update_all.sh releases
+    git add ${supported_versions}
+    git commit -m "Updating dockerfiles via update_dockerfiles_on_merge.sh script" -s
```

Comment:
The commit message should have a date and timestamp along with the PR number, something like
```
Timestamp: Auto-updating dockerfiles for PR #"${PR_NUMBER}"
```

Reply:
Added custom timestamp : `git commit -m "$(date '+%d-%m-%Y %H:%M:%S') : Auto-updating dockerfiles for PR #${PR_NUM}" -s`